cartridge/confapplier.lua

```diff
@@ -504,11 +504,21 @@ local function boot_instance(clusterwide_config)
             cartridge_schema_upgrade(clusterwide_config)
         end
 
-        BoxError:pcall(
-            box.schema.user.passwd,
-            username, password
-        )
+        local user = box.space[box.schema.USER_ID].index.name:get(username)
+        local change_passwd = true
+        if user ~= nil then
+            local old_passwd = user[5]['chap-sha1']
```

Comment:
What's 5?

Reply:
Even if there's no built-in constant, add a comment (preferably with a link to the code or spec)